README.md

```diff
@@ -58,28 +58,34 @@ If you would like to experiment with running this code against LabKey Server it
 
 #### Deploy Using Experimental Build
 
+:warning: This will modify files in your git enlistment. Double check your commits and be sure to revert and changes when you are done experimenting!
+
 To get started edit the `<labkey root>/server/modules/core/build.js` file and set `USE_LABKEY_API` to `true`.
 
 ```js
 const USE_LABKEY_API = true;
 ```
 
-Next, you can either perform a Gradle clean build or run the node portion of the build directly:
+Next, perform a Gradle build from the top of your LabKey enlistment:
 
 ```sh
-# From <labkey root>/server/modules/core
-npm install
-node build.js
+# From <labkey root>
+./gradlew deployApp
 ```
 
-Now you can startup the server and the code for the APIs will be supplied from this package.
+Once, the build is complete you can startup the server and the code for the APIs will be supplied from this package. This can be verified
```

Comment:
```suggestion
Once the build is complete, you can start the server and the code for the APIs will be supplied from this package. This can be verified
```

Reply:
"Once," lol. Me type good.